.github/pull_request_template.md

```diff
@@ -0,0 +1,51 @@
+<!--
+Thank you for contributing to this repository!
+
+You can help us review your changes by answering these questions.
+They are all optional, but the more information you provide the easier it will be for us to review your changes.
+Everyone here is a volunteer, so please help us out if you can.
+
+If you want to discuss anything Jupyter related, or to meet other users and developers, please say hello on https://discourse.jupyter.org/ .
```

Comment:
Should we add in this link as well? https://discourse.jupyter.org/t/introduce-yourself/17/

Reply:
Could do, though since we've got the welcome bot maybe we could add the introduce-yourself link there, perhaps with a stronger emphasis on "say hello", and just have the general link here?

Reply:
I agree with this but would actually reverse the suggestion. This is a PR template so folks using it are probably comfortable/familiar enough to make a contribution - we would just like to get to know them better (hence "say hello"). Whereas the bot's job is to orient newcomers who may be more likely to open a support request (hence general link). What do you think?

Also I really am splitting hairs here, so feel free to pass :)